Integrations/dnstwist/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Added subtype to integration yaml.
```

Comment:
If this is the only change you can leave the release notes empty. Simply put `-`